### PR TITLE
[simd] Add spec tests for f64x2 comparison ops

### PR DESCRIPTION
**Pull request by Honry**

### CODE REVIEW

**Comment by Honry** [comment]
@AndrewScheidecker, PTAL, thanks!

File: Test/spec/simd/meta/simd_f64x2_cmp.py [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+
+        for case in unary_test_data:
+            cases.append(self.single_unary_test(case))
+
+        self.get_unknow_operator_case(cases)
+
+        return '\n'.join(cases)
+
+    def get_unknow_operator_case(self, cases):
+        """Unknow operator cases.
```
Comment (AndrewScheidecker) [inline Test/spec/simd/meta/simd_f64x2_cmp.py]: s/unknow/unknown

Comment (Honry) [reply Test/spec/simd/meta/simd_f64x2_cmp.py]: Done, thanks!

**Comment by AndrewScheidecker** [comment]
Thanks!

**Pull request merged**